src/Form/Transformer/RestoreRolesTransformer.php

```diff
@@ -1,78 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-/*
- * This file is part of the Sonata Project package.
- *
- * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\UserBundle\Form\Transformer;
-
-use Sonata\UserBundle\Security\EditableRolesBuilder;
-use Symfony\Component\Form\DataTransformerInterface;
-
-class RestoreRolesTransformer implements DataTransformerInterface
```

Comment:
Isnt this a BC break

Reply:
Really dont know :/

Reply:
Looks like BC, what if someone was extending using a different transformer for that? Would it break his App?

Reply:
yes it is, please keep the class as it is, and throw a deprecation warning, also keep the existing test cases for this class and add `@group legacy` to them.

We will remove the class in the next major version.